client/src/components/pages/ViewUsers/index.js

```diff
@@ -1,21 +1,85 @@
-import React, { Component } from "react";
-import {Link} from 'react-router-dom'
+import axios from 'axios';
+import React, {	Component} from "react";
+import {	Link} from 'react-router-dom'
```

Comment:
Remove this space here, run `eslint:fix`.

Reply:
done